apps/src/lib/kits/maker/ui/EligibilityChecklist.jsx

```diff
@@ -95,6 +95,14 @@ export default class EligibilityChecklist extends React.Component {
     });
   };
 
+  schoolHighNeedsStatusMessage = schoolHighNeedsEligible => {
+    return schoolHighNeedsEligible === true ? (
```

Comment:
Curious: Does it need to be explicitly `true` or would truthy be enough?

Reply:
mmm truthy would probably be enough? The argument should always either be `true` or `false` in the one place that this method is called, so I figured it was better to be explicit about it. Not sure if that's the right call, though.